tests/tasks/test_testing.py

```diff
@@ -71,3 +79,82 @@ def test_dummy_timestamp_feature_task(dummy_eopatch: EOPatch):
 
     eopatch3 = task.execute(dummy_eopatch.copy(), seed=11)
     assert eopatch1 != eopatch3
+
+
+@pytest.mark.parametrize(
+    ("feature_type", "shape", "dtype", "min_value", "max_value"),
+    [
+        (FeatureType.DATA, (10, 20, 21, 5), np.float32, -3, -1.5),
+        (FeatureType.DATA_TIMELESS, (83, 69, 1), int, -2, 5),
+        (FeatureType.MASK, (10, 4, 16, 7), np.int8, -3, 7),
+        (FeatureType.LABEL, (10, 271), bool, 0, 1),
+    ],
+)
+@pytest.mark.parametrize("seed", [1, 22, 9182])
+def test_generate_raster_feature_task_uniform(dummy_eopatch, feature_type, shape, dtype, min_value, max_value, seed):
+    dummy_eopatch.timestamps = ["2011-08-12"] * 10
```

Comment:
would it be better to use the info from shape here?

```suggestion
    dummy_eopatch.timestamps = ["2011-08-12"] * shape[0]
```

Reply:
Ah, not in all the cases..